pkgs/applications/networking/browsers/vivaldi/default.nix

```diff
@@ -16,6 +16,7 @@
 , enableWidevine ? false, widevine-cdm ? null
 , commandLineArgs ? ""
 , pulseSupport ? stdenv.isLinux, libpulseaudio
+, libkrb5
```

Comment:
Perhaps add a `kerberosSupport` flag here?

Reply:
You definitely can set this to true.

Reply:
Thanks, done.